_sass/base/_variables.scss

```diff
@@ -17,7 +17,8 @@ $border-color: rgba(0, 0, 0, 0.1); // rgba recommended if using feature images
 
 // Typography colours
 $text-color: #383838;
-$link-color: $theme-color; // #1ABC9C;
+//$link-color: $theme-color; // #1ABC9C;
```

Comment:
Similarly, we can use git to look at an older version of the code. So we can remove the old version instead of commenting it out.